modules/stochastic_tools/test/tests/reporters/AISActiveLearning/gold/ais_al.json

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
This is just a suggestion: in the future (maybe here too) please create smaller files for golding results. I think json is especially dangerous in this sense. It can generate a lot of text even for simple cases.

Reply:
Yes, you're right. The test here also tests the `AdaptiveImportanceStats` object for Pf and Cov calc. We don't get a value for these stats until we run at least 15-20 samples :)